idofront-util/src/main/kotlin/com/mineinabyss/idofront/plugin/Plugins.kt

```diff
@@ -22,7 +22,10 @@ object Plugins {
         getOrNull<T>()?.let { isEnabled(it) } ?: false
 
     /** Checks if a [plugin] exists and is enabled. */
-    fun isEnabled(plugin: Plugin) = Bukkit.getPluginManager().isPluginEnabled(plugin)
+    fun isEnabled(plugin: Plugin) = runCatching { Bukkit.getPluginManager().isPluginEnabled(plugin) }.isSuccess
+
+    /** Checks if a [plugin] exists and is enabled. */
+    fun isEnabled(plugin: String) = Bukkit.getPluginManager().isPluginEnabled(plugin)
```

Comment:
Thanks for getting this one in, I've surprised myself recently about forgetting it